data-migration/migrate-aurora-tidb-from-snapshot.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Migrate Data from Amazon Aurora to TiDB
+summary: Learn how to migrate data from Amazon Aurora to TiDB using DB snapshot.
+---
+
+# Migrate Data from Amazon Aurora to TiDB
+
+This document describes how to migrate data from Amazon Aurora to TiDB. The migration process uses [DB snapshot](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Backups.html), which saves a lot of space and time.
+
+The whole migration has two processes:
+
+- Import full data to TiDB using TiDB Lightning
+- Replicate incremental data to TiDB using DM (optional)
+
+## Prerequisites
+
+- [Install Dumpling and TiDB Lightning](/migration-tools.md)
+- [Get the target database privileges required for TiDB Lightning](/tidb-lightning/tidb-lightning-faq.md#what-are-the-privilege-requirements-for-the-target-database).
+
+## Import full data to TiDB
+
+### Step 1. Export an Aurora snapshot to Amazon S3
+
+1. In Aurora, query the current binlog position by running the following command:
+
+    ```sql
+    mysql> SHOW MASTER STATUS;
+    ```
+
+    The output is similar to the following. Record the binlog name and position for later use.
+
+    ```
+    +------------------+----------+--------------+------------------+-------------------+
+    | File             | Position | Binlog_Do_DB | Binlog_Ignore_DB | Executed_Gtid_Set |
+    +------------------+----------+--------------+------------------+-------------------+
+    | mysql-bin.000002 |    52806 |              |                  |                   |
+    +------------------+----------+--------------+------------------+-------------------+
+    1 row in set (0.012 sec)
+    ```
+
+2. Export the Aurora snapshot. For detailed steps, refer to [Exporting DB snapshot data to Amazon S3](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_ExportSnapshot.html).
+
+After you obtain the binlog position, export the snapshot within 5 minutes. Otherwise, the recorded binlog position might be outdated and thus cause data conflict during the incremental replication.
+
+After the two steps above, make sure you have the following information ready:
+
+- The Aurora binlog name and position at the time of the snapshot creation.
+- The S3 path where the snapshot is stored, and the SecretKey and AccessKey with access to the S3 path.
+
+### Step 2. Export schema
+
+Because the snapshot file does not contain the DDL statements, you need to export the schema using Dumpling and create the schema in the target database using TiDB Lightning. If you want to manually create the schema, you can skip this step.
```

Comment:
```suggestion
Because the snapshot file from Aurora does not contain the DDL statements, you need to export the schema using Dumpling and create the schema in the target database using TiDB Lightning. If you want to manually create the schema, you can skip this step.
```